analytics.js

```diff
@@ -920,6 +920,9 @@ class Analytics {
     if (options && options.logLevel) {
       logger.setLogLevel(options.logLevel);
     }
+    if (options && options.setCookieDomain) {
```

Comment:
Do we need to provide any validation check for the value given in the `setCookieDomain` field?

Reply:
If an invalid domain is given be it in any form, we will fallback to present page hostname.